src/index.html

```diff
@@ -17,6 +29,82 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_list">
```

Comment:
The class attribute value should be `nav__list` instead of `nav_list` to follow the BEM naming convention.